### DISCUSSION TITLE
How could I update PyG to support heterogeneous graphs?

### DISCUSSION THREAD

**Discussion by YAndrewL** [pyg-team/pytorch_geometric#3084] [Q&A] [answered]
Hi~ I found that PyG has add some new functions for heterogeneous, I'm wondering how could I update this from version 1.7.2?

**Comment by rusty1s** [comment, marked as answer]
You can install via master to test out the new features:
```
pip install git+https://github.com/rusty1s/pytorch_geometric.git
```
A proper release will follow soon :)
